schema/config.model.schema

```diff
@@ -443,7 +443,7 @@
           "inputType": "Checkbox",
           "validators": [],
           "title": "Use strict mode?",
-          "help": "Uses ES6 modules. If AMD-style module definitions are required for plugins, turn this off"
+          "help": "Strict mode drops suport for AMD-style module definitions. If these are still required for any plugins, turn this setting off"
```

Comment:
You can use both AMD and ES modules in adapt whether or not strict mode is enabled.

Strict mode improves performance by tightly restricting the declaration of variables inside code blocks and closures which may cause some older code to stop working. Please leave this at true unless you are experiencing issue with old plugins or third party libraries.

Reply:
Thanks Oli